docs/admin_guides/authorizations.md

```diff
@@ -0,0 +1,93 @@
+Otherwise, everything else is exactly the same.
+
+* **Scope**
+A scope is a domain grouping different permissions related to the same topic. Permissions from certain scope can be limited (ie: character or corporation).
+
+* **Permission**  
+A Permission is an attribute that is assigned to a *Role*. It grants access based on the specific permission.
+
+* **Role**  
+A Role is simply a collection of permissions. Users get assigned a roles and inherit the permissions granted by that role.
+A user cannot be given a raw permission. Permissions can only be granted by creating a Role, assigning permissions to the Role and granting the role to a user.
+
+* **Filters**
+Permissions from certain scope can received filters. That simply mean the granted permission is limited to certain conditions.
+As an example, you may want to limit the asset permission from character scope to only a selection of character.
+
+* **Entity**  
+An entity is something on which the permission will be applied. It can be a Character, a Corporation or anything else.
+
+## Interface
+
+To manage roles, you must go into Settings from SeAT sidebar, then choose Access Management.
+
+!!! info
+    To be able to manage SeAT roles, you must be authenticated as an administrator user.
+    Administrators can be managed in the users list, located in Settings.
+
+### Role card
+
+The Role card is compounds of two main area.
+
+![Role Card](../img/authorization.png)
+
+Left pane contains basic information. It allows you to provide a name, a description and upload an optional logo.
+Those information can be sync with any third party platform using the API.
+
+!!! tip
+    Provided logo are stored into database - so you don't need to worry when moving your installation.
+
+Right pane contains role settings. The pane is split in two tabs - first tab is showing the role permissions definitions.
+Second pane, the members one, gives you the current role members and related management actions.
+
+### Role definition
+
+#### Role permissions
+
+![Role Permissions](../img/authorization_permissions.png)
+
+The role permissions tab is built using a navigation bar, which is showing all available scopes - and the list of permissions from active scope.
+Use the scope navigation to show permissions related to each scope.
+
+Each permission block is structured as follow:
+- a checkbox: if you check it, you mark the permission to be granted by the role
+- an optional icon: it is visual only and help to pair some permission inside a same scope. It might be useful for large scope like character or corporation to distinguish permission kind. Options include `military`, `financial` and `industrial`.
+- a user friendly name
+- an optional description: this is mostly a text explaining the permission purpose
+- an optional limits button: it will help you to filter/restrict the permission to only a certain population. Only character and corporation scope permissions can be filtered.
+
```

Comment:
```suggestion
- an optional limits button: it will help you to filter/restrict the permission to only a certain population. Only character and corporation scope permissions can be filtered.

!!! warning

    In SeAT 3, leaving the affiliations of a role empty, meant that the role would apply to no relevant entities. It had no effect. This has changed in SeAT 4. If you give a role a permission with no filter, this permission will apply globally to all entities. For example giving the Corporation Sheet permission to a role with no filter means that the members of the role will be able to see the corporation sheets of every corporation on the server.
```

@warlof can you make sure this renders correctly in mkdocs, the github web editor shows it weirdly.